lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -275,4 +275,21 @@ export class AccountEntity {
             entity.hasOwnProperty("authorityType")
         );
     }
+
+    /**
+     * Helper function to determine whether 2 accounts are equal
+     * Used to avoid unnecessary state updates
+     * @param arrayA 
+     * @param arrayB 
+     */
+    static accountInfoIsEqual(accountA: AccountInfo | null, accountB: AccountInfo | null): boolean {
+        if (!accountA || !accountB) {
+            return false;
+        }
+        return (accountA.homeAccountId === accountB.homeAccountId) && 
+            (accountA.localAccountId === accountB.localAccountId) &&
+            (accountA.username === accountB.username) &&
+            (accountA.tenantId === accountB.tenantId) &&
+            (accountA.environment === accountB.environment);
```

Comment:
Shouldn't we respect environment aliases here?

Reply:
For now this is just an exact match. It is only being used to clear the active account. If we want to add environment aliasing for other scenarios we can, but I think for logout it is fine.

Reply:
okay!